template.json

```diff
@@ -359,6 +360,14 @@
     "energyPointToken": {
       "active": false,
       "holder": ""
+    },
+    "shipLogbook": {
+      "destination": "",
+      "segment": "",
+      "route": "",
+      "danger": "",
+      "checkout": "",
+      "description": ""
```

Comment:
Sure I guess. Would be the same result since the base is defined as "". But for consistency reasons you're right.
![image](https://user-images.githubusercontent.com/36819852/174680574-1d716749-c97d-4e53-968a-9802a30cbc59.png)